app/src/main/java/fr/groggy/racecontrol/tv/ui/season/browse/SeasonBrowseFragment.kt

```diff
@@ -49,6 +69,46 @@ class SeasonBrowseFragment : BrowseSupportFragment(), OnItemViewClickedListener
         adapter = eventsAdapter
     }
 
+    /**
+     * Function that will load a [Session.largePictureUrl] as a background image when a card is selected
+     */
+    private fun initializeBackground() {
+        val backgroundManager = BackgroundManager.getInstance(activity).apply {
+            attach(activity?.window)
+        }
+
+        val metrics = DisplayMetrics()
+        requireActivity().windowManager.defaultDisplay.getMetrics(metrics)
```

Comment:
Hm, this could actually mean that you have a 1080p image loaded in a custom target of 720p (in some TVs), which, might be fine but how does it looks?